crypto/x509v3/v3_addr.c

```diff
@@ -1180,9 +1184,13 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
     int i, j, ret = 1;
     X509 *x;
 
-    OPENSSL_assert(chain != NULL && sk_X509_num(chain) > 0);
-    OPENSSL_assert(ctx != NULL || ext != NULL);
-    OPENSSL_assert(ctx == NULL || ctx->verify_cb != NULL);
+    if (!ossl_assert(chain != NULL && sk_X509_num(chain) > 0)
```

Comment:
Well, what I tried to say is that I agree with Rich, i.e. would be in favour of removal (with rationale that of three checks only first is valuable, but even if it, first one, is omitted the condition is covered few lines below). Oh well...